src/main/java/wooteco/subway/exception/SubwayControllerAdvice.java

```diff
@@ -0,0 +1,67 @@
+package wooteco.subway.exception;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.dao.DuplicateKeyException;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import wooteco.subway.exception.dto.ErrorResponse;
+import wooteco.subway.path.application.InvalidPathException;
+
+import java.util.Arrays;
+
+@RestControllerAdvice
+public class SubwayControllerAdvice {
+
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @ExceptionHandler(AuthorizationException.class)
+    public ResponseEntity<ErrorResponse> handleAuthorizationException(AuthorizationException e) {
+        return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(new ErrorResponse(e.getMessage()));
+    }
+
+    @ExceptionHandler(StationException.class)
```

Comment:
지금 `return ResponseEntity.badRequest().body(new ErrorResponse(e.getMessage()));` 형태로 리턴하는 handler들이 많은데, 아래처럼 한번에 처리해주면 어떨까요? 
나중에 다른 처리가 필요할때 분리해도 될 것 같아요! 
```suggestion
    @ExceptionHandler({StationException.class, LineException.class, ... })
```